service/src/main/java/eu/europa/ec/fisheries/uvms/user/service/bean/OrganisationServiceBean.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Developed by the European Commission - Directorate General for Maritime
+ * Affairs and Fisheries © European Union, 2015-2016.
+ *
+ * This file is part of the Integrated Fisheries Data Management (IFDM) Suite.
+ * The IFDM Suite is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or any later version.
+ * The IFDM Suite is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details. You should have received a copy of the GNU General Public
+ * License along with the IFDM Suite. If not, see http://www.gnu.org/licenses/.
+ */
+package eu.europa.ec.fisheries.uvms.user.service.bean;
+
+import eu.europa.ec.fisheries.uvms.user.service.OrganisationService;
+import eu.europa.ec.fisheries.uvms.user.service.converter.OrganizationMapper;
+import eu.europa.ec.fisheries.uvms.user.service.dao.OrganisationDao;
+import eu.europa.ec.fisheries.uvms.user.service.dto.OrganisationDto;
+import eu.europa.ec.mare.usm.information.entity.OrganisationEntity;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+
+@ApplicationScoped
+@Transactional
+public class OrganisationServiceBean implements OrganisationService {
+
+    @Inject
+    private OrganisationDao organisationDao;
+
+    @Inject
```

Comment:
Please use constructor injection! Reasons:

1. The class is usable in environments without CDI (e.g. you can test it with `new`)
2. It is easier to spot components with too many dependencies. This hints that the class is doing too much and probably violates the single responsibility principle.